### ISSUE TITLE
New conversion to square km for area is incorrect

### ISSUE THREAD

**Issue by otherjason** [andrewharvey/geojson-polygon-labels#1]
In the previous release, the area was specified in square meters. In 1.0.2, that was intended to be changed to square kilometers; this conversion [is done here](https://github.com/andrewharvey/geojson-polygon-labels/blob/master/bin/geojson-polygon-labels#L82). However, this isn't correct; to convert square meters to square kilometers, you need to divide by 1 million (1000 * 1000), not by 1000.

**Comment by andrewharvey** [comment]
Thanks for flagging this @otherjason. I'm still very new to publishing modules to npm, versioning and releasing and I've made a few failures here.

To fix this I'll
- [x] add a changelog so people can see what changes have been made version to version
- [x] use semver so you know which changes are breaking
- [x] tag releases in github
- [x] revert back to sq m to avoid the breaking change

- [ ] rethink the units. The reason I did /1000 is I was finding the numbers to be too large and trying to count 0's was a pain. But you're right my basic math failed here. Notation like `1.234e-5` would be helpful, I might add another option which just has the base 10 exponent number, and keep _area in sq meters.
